l2-contracts/contracts/bridge/interfaces/IL2NativeTokenVault.sol

```diff
@@ -21,11 +22,11 @@ interface IL2NativeTokenVault is IL2AssetHandler {
         uint256 amount
     );
 
-    event L2TokenBeaconUpdated(address l2TokenBeacon, bytes32 l2TokenProxyBytecodeHash);
+    event L2TokenBeaconUpdated(address indexed l2TokenBeacon, bytes32 indexed l2TokenProxyBytecodeHash);
 
     function l2Bridge() external view returns (IL2SharedBridge);
 
-    function tokenAddress(bytes32 _assetId) external view returns (address);
+    function tokenAddress(bytes32 assetId) external view returns (address);
```

Comment:
Is the underscore missing on purpose? _assetId

Reply:
also a mapping